### DISCUSSION TITLE
Why can't some markdown files on GitHub be previewed?

### DISCUSSION THREAD

**Discussion by Kukukukiki192** [community/community#73381] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The markdown file in Figure 1 can be previewed like READMA.md, but Figure 2 can't find the preview button. Is this a bug in GitHub?
![](https://img-blog.csdnimg.cn/img_convert/e7d44303c81de36254d6d764f2a9f43d.png)
![](https://img-blog.csdnimg.cn/img_convert/3fd66e7aac7ddb09ddacb6d27f4ebb56.png)

**Comment by b8zeek** [comment]
Good catch, that is weird! Seems like a GitHub issue. 🤔 

In the meantime, you can copy contents of the file, create a gist and preview it. Example below:
https://gist.github.com/bejzik8/96a21f2694a57eecc2076b891198d132

Take care! 🍺 

🍀

**Reply by Kukukukiki192** [reply]
Exm. This link shows 404.

**Reply by b8zeek** [reply]
It's because the gist is deleted, I assumed you will take a look but you didn't reply for two weeks.

Here's the new link:

https://gist.github.com/bejzik8/c481de7e2484cc7bc7a02f0a560d810e

🍀

**Reply by Kukukukiki192** [reply]
OK. I can see it now. Thx!

**Reply by b8zeek** [reply]
You're welcome. ☺️

🍀
